src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Can you use this annotation for classes? Or only for field? [See here ](https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/Target.html)
add target to all custom annotations

Reply:
it's a good practice to add `Target` for all custom annotations. `Inject` is not an exclusion :)